src/main/java/com/nekomaster1000/infernalexp/client/entity/model/GlowsilkMothModel.java

```diff
@@ -0,0 +1,78 @@
+package com.nekomaster1000.infernalexp.client.entity.model;// Made with Blockbench 3.8.2
+// Exported for Minecraft version 1.15
+// Paste this class into your mod and generate All required imports
```

Comment:
```suggestion
package com.nekomaster1000.infernalexp.client.entity.model;
```
Can we remove these comments?